tree/dataframe/inc/ROOT/RDF/ActionHelpers.hxx

```diff
@@ -501,7 +501,19 @@ public:
       return std::make_unique<RMergeableFill<HIST>>(*fObjects[0]);
    }
 
-   std::string GetActionName() { return "Fill"; }
+   // if the fObjects vector type is derived from TObject, return the name of the object
+   template <typename T = HIST, typename std::enable_if<std::is_base_of<TObject, T>::value>::type * = nullptr>
+   std::string GetActionName()
+   {
+      return "TObject<BR/>" + std::string(fObjects[0]->GetName());
```

Comment:
I don't like that we show "TObject" for everything (because we are never using an actual TObject rather some derived class). Would be nicer if we could print the actual type of the class with e.g. `fObjects[0]->IsA()->GetName()`

Reply:
Agreed, I am changing to it `return std::string(fObjects[0]->IsA()->GetName()) + "<BR/>" + std::string(fObjects[0]->GetName())`